### DISCUSSION TITLE
can one add back to templetar when expanding through it a {{value}}?

### DISCUSSION THREAD

**Discussion by zeltak** [chhoumann/quickadd#452] [General]
Hi all I have this quick add capture
![CleanShot 2023-04-08 at 16 52 40](https://user-images.githubusercontent.com/498410/230724902-4a2acae5-da59-416a-a1ba-29ca67b2b60d.png)

im would love to have a filed in my specific template template that uses the `{{VALUE:name}}` I enter when the capture starts to use in my template ie

```
---
tags: 
- people
status: 
rating: 
location: [0.0,0.0]
counter: 
created:  <% tp.date.now() %>
url:
---

## <span style="background:#affad1">tasks</span>
tasks
    not done
    (tag includes. {{VALUE:name}})
    sort by priority
    sort by due  

```

is that even possible :) 

thx so much

z

**Comment by chhoumann** [comment]
Hi @zeltak!

I'm not sure that I understand the workflow you're after, but you can use `{{VALUE:name}}` in the template - it should get filled it when QuickAdd creates the file.

If this isn't quite what you're after, can you then clarify the workflow you're looking for?
What would you like to happen?

**Comment by zeltak** [comment]
thx @chhoumann !

sorry for not being clear. I want the `{{VALUE:name}}` to go back to the template template based on the input I give first in quickadd

ie I have this template below, I want to add the  `{{VALUE:name}}` into the templater template itself  when it expands?


```
---
tags: 
- people
- meeting 
status: 
rating: 
location: 
counter: 
created:  <% tp.date.now() %>
url:
---

## <span style="background:#affad1">tasks</span>

tasks
    not done
    (tag includes  {{VALUE:name}} )
    sort by priority
    sort by due  


## meetings

```

so above when I enter the  `{{VALUE:name}}` name `ADAM` the expanded template will have the line

```
   (tag includesADAM )
```

 does that make sense?

thx so much form you AWESOME plugin!

best

Z

**Reply by chhoumann** [reply]
Thank you for clarifying!
This should be possible when you just write {{VALUE:name}} in the template. The name you enter should get inserted any place you write that. 

LMK if you encounter any issues with that

**Comment by zeltak** [comment]
ahh perfect !!!

thx so much!!

Z
